### ISSUE TITLE
Can't edit global fuel types

### ISSUE THREAD

**Issue by mhbates** [APDevTeam/Movecraft#453]
**Describe the bug**
I tried adding global fuel types per the wiki page https://github.com/APDevTeam/Movecraft/wiki/Fuel-Types, but it doesn't seem to have an effect.

Workaround for now is to add the fuel types to the individual ship configs. Note that when this is done you also need to re-add the default fuels which are coal and coal block. This workaround is fine but obviously it'd be better if this didn't have to be replicated for each ship type.

**Versions (please complete the following information):**
 - Movecraft: 8 alpha 4
 - Java 16 (OpenJDK 64-Bit Server VM 16.0.1+9)
 - Minecraft 1.16.5 (Airplane)

**Comment by TylerS1066** [comment]
It appears a contributor outside of the APDevTeam created that wiki page about a feature not available on our version.  That wiki page has been removed.

If you would like to change this into a suggestion to add the global fuel types, we can do so.  Otherwise, I'll be closing this in a few days.

**Comment by mhbates** [comment]
Yes, please, I think this would be a good suggestion. I'm guessing some other people might want universal fuels as well.
